skbio/stats/composition.py

```diff
@@ -802,6 +803,274 @@ def centralize(mat):
     return perturb_inv(mat, cen)
 
 
+<<<<<<< HEAD
```

Comment:
@mortonjt still here. Please do check through the files

Reply:
let's wait for all of the tests to pass first before reviewing.